ubcpi/static/html/ubcpi.html

```diff
@@ -158,6 +160,8 @@ <h2 id="pi-question-h" class="question-text">Question</h2>
                 <div class="detailed-solution" id="detailed-solution">
 
                     <h2 id="correct-option-head" ng-if="options.length > rc.correct_answer">Correct Answer</h2>
+                    <h2 id="correct-option-head" ng-if="!rc.correct_rationale.text"></h2>
```

Comment:
Yes; the way things are currently set up, if we remove this check, a heading will be inserted (we don't want this) if there is no rationale.